src/shapes/object.class.js

```diff
@@ -974,9 +974,7 @@
      * @returns {number}
      */
     getTotalAngle: function () {
-      return this.group ?
-        fabric.util.qrDecompose(this.calcTransformMatrix()).angle :
-        this.angle;
+      return fabric.util.qrDecompose(this.calcTransformMatrix()).angle;
```

Comment:
is there any reason why we want this to not use shortcuts?

Reply:
this was something I saw when I merged  #7726 so I added it to there....
Shortcuts should be used